src/types/Person.ts

```diff
@@ -0,0 +1,6 @@
+import { PersonFromServer } from './PersonFromServer';
+
+export interface Person extends PersonFromServer {
+  mother: PersonFromServer | undefined,
+  father: PersonFromServer | undefined,
```

Comment:
usually objects are `null` if we have no data
```suggestion
  mother: PersonFromServer | null,
  father: PersonFromServer | null,
```